### DISCUSSION TITLE
跨页表的分页问题

### DISCUSSION THREAD

**Discussion by zhzh2001** [Keldos-Li/typora-latex-theme#52] [Q&A] [answered]
还是分页问题。创建了一个很长的表格，但表格开始似乎会强制另起一页，导致半页都是空白的。尝试了 `page-break-before`，但由于表格是用 markdown 写的，无法写入对应的 HTML 标签，在表格前增加 `<p style="page-break-before:avoid"></p>` 也没有用……

![image](https://user-images.githubusercontent.com/24852958/143669313-79c64a8b-1455-44ad-8899-84f705eda0be.png)

**Comment by Keldos-Li** [comment, marked as answer]
_(:з」∠)_因为做不了续表功能所以优先考虑禁止断开表格的。如果想要避免太长的空白，可以试试先把这行去掉：
<img width="537" alt="image" src="https://user-images.githubusercontent.com/23137268/143690569-9cd04ce0-6388-4ea4-bc6a-0f845331c7be.png">

**Reply by RalXYZ** [reply]
确实！删了这一行 CSS 后，导出的 PDF 变成了这个样子，感觉还行：  

![image](https://user-images.githubusercontent.com/19510622/144089007-69ac221b-bdb9-4eed-acf5-b5d15f3ab764.png)
